client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -610,7 +610,7 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 	const displayedPlansProperties = useMemo(
 		() =>
 			( planProperties ?? [] ).filter(
-				( { planName } ) => ! ( planName === PLAN_ENTERPRISE_GRID_WPCOM )
+				( { planName, isVisible } ) => isVisible && ! ( planName === PLAN_ENTERPRISE_GRID_WPCOM )
```

Comment:
`displayedPlansProperties` is used to determine what plans to show in the comparison table on smaller screens and it gets changed each time swap plans by clicking on the name (or cycle between screen sizes).